tests/automount/test_automountlocation.yml

```diff
@@ -0,0 +1,104 @@
+---
+- name: Test automountlocation
+  hosts: ipaserver
+  become: true
+  gather_facts: false
+
+  tasks:
+  - name: ensure automountlocation TestLocations are absent before testing
+    ipaautomountlocation:
+      ipaadmin_password: SomeADMINpassword
+      name:
+      - TestLocation_01
+      - TestLocation_02
+      state: absent
+
+  - name: ensure empty automountlocation does nothing
+    ipaautomountlocation:
+      ipaadmin_password: SomeADMINpassword
+      name: []
+      state: present
+    register: result
+    failed_when: not result.failed
+
+  - name: ensure empty automountlocation does nothing on absent
+    ipaautomountlocation:
+      ipaadmin_password: SomeADMINpassword
+      name: []
+      state: absent
+    register: result
+    failed_when: not result.failed
```

Comment:
Please, change the `failed_when` test to `failed_when: not result.failed or "At least one location must be provided" not in result.msg`, as stated before.